packages/components/psammead-story-promo/src/TextStyles.jsx

```diff
@@ -40,6 +40,7 @@ ${({ displayImage }) => !displayImage && `grid-column: 1 / span 6;`}
 
 @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
   padding-top: ${GEL_SPACING};
+  ${({ displayImage }) => !displayImage && `padding-top: 0;`}
```

Comment:
Thanks for fixing this, Simon!
Please can we use a ternary on L42 as opposed to adding an additional `padding-top` property to our CSS?